recipes/tinyarray/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "tinyarray" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://gitlab.kwant-project.org/basnijholt/tinyarray/repository/archive.tar.gz?ref=v{{ version }}
```

Comment:
This is not the main repository, I've opened a [pull request](https://gitlab.kwant-project.org/kwant/tinyarray/merge_requests/6).